server/routes/api/reviews.js

```diff
@@ -111,4 +111,34 @@ router.get("/requests/:id", authenticate, async (req, res) => {
   }
 });
 
+router.put("/:id", authenticate, async (req, res) => {
+  const requestId = req.params.id;
+  const { messageId, code, comments, authorId } = req.body;
```

Comment:
Should check if you're receiving the required inputs and return status 400 accordingly.